src/SlabSeaIceModels/slab_sea_ice_model.jl

```diff
@@ -46,6 +46,7 @@ struct SlabSeaIceModel{GR, CL, TS, IT, IC, ST, IS, U, STF, TBC, CF, P, MIT, A} <
     ice_consolidation_thickness :: MIT
     # Numerics
     advection :: A
+    architecture
```

Comment:
```suggestion
    architecture :: Arch
```

This needs a type parameter to be concrete

but you should be able to get the `architecture` from the `grid`?